src/dashboard/Dashboard.js

```diff
@@ -226,7 +226,7 @@ class Dashboard extends React.Component {
           <Route path='config' component={Config} />
           <Route path='api_console' component={ApiConsole} />
           <Route path='migration' component={Migration} />
-          <Redirect from='push' to='/apps/:appId/push/activity/all' />
+          <Redirect from='push' to='/apps/:appId/push/new' />
```

Comment:
Push status aren't saved on the server yet so there is no data to load. Once we have some data to load, we'll add the index page back. I figured this was a find temporary solution.